skyportal/handlers/api/source.py

```diff
@@ -943,6 +954,16 @@ class Validator(Schema):
             source_query = source_query.filter(Source.saved_at <= saved_before)
         if saved_after:
             source_query = source_query.filter(Source.saved_at >= saved_after)
+        if created_or_modified_after:
+            created_or_modified_date = arrow.get(
```

Comment:
This will throw an uncaught exception if `created_or_modified_after` is not in a proper format. Suggest gracefully catching this exception and responding accordingly.

Reply:
Agreed -- done.